cpp/examples/deck.gl/stdinout.cpp

```diff
@@ -35,7 +35,7 @@ int main(int argc, char** argv) {
   registerJSONConvertersForDeckLayers(jsonConverter.get());
 
   auto deckProps = make_shared<Deck::Props>();
-  auto deck = unique_ptr<Deck>(new Deck(deckProps));
+  auto deck = make_unique<Deck>(deckProps);
```

Comment:
Should we have a default argument so we don't need to create an empty props object?